base/linalg/lapack.jl

```diff
@@ -657,6 +667,7 @@ for (gesvx, elty) in
         function gesvx!(fact::Char, trans::Char, A::StridedMatrix{$elty},
                AF::StridedMatrix{$elty}, ipiv::Vector{BlasInt}, equed::Char,
                R::Vector{$elty}, C::Vector{$elty}, B::StridedVecOrMat{$elty})
+            @chktrans
             lda, n    = size(A)
```

Comment:
It is, but it is also a bit uncertain how robust our linear algebra code is for `SubArrays`. Unfortunately, we cannot really capture that with CodeCoverage.